twine/sdist.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Daniele Nicolodi
```

Comment:
Same here.

Also, I believe the old copyright notices are:
A. Unnecessary
B. Should probably get removed at some point

Reply:
Same reply as above. I just followed the pattern in all other source files in `twine`.